### DISCUSSION TITLE
use wrong firmware

### DISCUSSION THREAD

**Discussion by hczhnssb** [timschneeb/GalaxyBudsClient#222] [Q&A] [answered]
My headset is bus pro, but I accidentally used the firmware of buds plus, and now I can't flash the firmware back to the firmware of bud pro. I found that you seem to have made restrictions in the galaxy buds manager.

**Comment by timschneeb** [comment, marked as answer]
Are your earbuds even working/connecting after flashing the wrong firmware?

The option to select a custom firmware file has no restrictions in the app at all (firmware files do not contain any information about the model they are meant for).
However, the built-in firmware list will only show the correct firmware files for the currently connected model.

If you're lucky, the earbuds still work but report themselves to my application as a pair of Buds+, which causes the firmware flasher to hide the Buds Pro firmware versions from the list.

1. Download the latest Buds Pro firmware here: https://timschneeberger.me/stash/FOTA_R190XXU0AUD5.bin
2. In the app, select 'Advanced > Select firmware from disk...' and select the firmware file.

With some luck, you can repair it.
